.ci/jobs/fleet-server.yml

```diff
@@ -13,7 +13,7 @@
         discover-pr-forks-trust: permission
         discover-pr-origin: merge-current
         discover-tags: true
-        head-filter-regex: '^(?!update-stack-version).*$'
+        head-filter-regex: '^(?!update-.*-version).*$'
```

Comment:
`update-stack-version` is the one for the automation regarding the elastic stack versions.
`update-go-version` is the one to bump the current go-version to the latest stable one.